ferrilata-devilline/Controllers/PitchController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using ferrilata_devilline.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ferrilata_devilline.Controllers
+{
+
+    [Route("api")]
+    public class PitchController : Controller
+    {
+        public PitchController()
+        {
+        }
+
+        [HttpPost("post/pitch")]
+        public IActionResult PostPitch([FromBody] Pitch NewPitch)
+        {
+            var request = Request;
```

Comment:
no need to create variables for this, you can access `Request.Headers` from the expression itself, it's only a little bit longer, but still shorter than creating new variables for them

Reply:
done